src/xcode/ENA/ENA/Source/Workers/Store/SecureStore.swift

```diff
@@ -79,7 +79,7 @@ final class SecureStore: Store {
 	}
 
 	var tan: String? {
-		get { kvStore["tan"] as String? ?? "" }
```

Comment:
❤️

Reply:
> ❤️
@ChristianKienle 
the `tan` is never nil?